pypesto/optimize/ess/sacess.py

```diff
@@ -41,8 +41,10 @@
 class SacessOptimizer:
     """SACESS optimizer.
 
-    A shared-memory-based implementation of the SaCeSS algorithm presented in
-    :footcite:t:`PenasGon2017`. Multiple processes (`workers`) run
+    A shared-memory-based implementation of the
+    `Self-Adaptive Cooperative Enhanced Scatter Search` (SaCeSS) algorithm
+    presented in :footcite:t:`PenasGon2017`. This is a meta-heuristic for
+    global optimization. Multiple processes (`workers`) run
```

Comment:
```suggestion
    global optimization. Multiple processes (``workers``) run
```

Reply:
I was undecided there. I used double-backticks before, but my impression was that more often, single backticks were used when referring to argument names. Checking again, it seems like numpy and sphinx also use a wild mix... Maybe we should write down the convention we want to follow there.

Reply:
Ah, I was a little overzealous here I think. "workers" is not referring to a variable name, right? I only meant to add double backticks if it was a variable name. Then italics looks good here :+1: